integration-tests/tests/api/audit-logs/audit-log-record.spec.ts

```diff
@@ -0,0 +1,194 @@
+  expect(auditLogs?.[0].node.eventTime).toBeDefined();
+});
+
+const ExportAllAuditLogs = graphql(`
+  mutation exportAllAuditLogs($selector: OrganizationSelectorInput!, $filter: AuditLogFilter!) {
+    exportOrganizationAuditLog(selector: $selector, filter: $filter) {
+      ok {
+        url
+      }
+      error {
+        message
+      }
+      __typename
+    }
+  }
+`);
+
+test.concurrent(
+  'Try to export Audit Logs from an Organization with unauthorized user',
+  async () => {
+    const { createOrg, ownerToken } = await initSeed().createOwner();
+    const firstOrg = await createOrg();
+    const secondOrg = await initSeed().createOwner();
+    const secondToken = secondOrg.ownerToken;
+
+    const exportAuditLogs = await execute({
+      document: ExportAllAuditLogs,
+      variables: {
+        selector: {
+          organizationSlug: firstOrg.organization.id,
+        },
+        filter: {
+          startDate: '1960-12-31T22:00:00.000Z',
+          endDate: '2023-12-31T22:00:00.000Z',
+        },
+      },
+      token: secondToken,
+    });
+
+    expect(exportAuditLogs.rawBody.data?.exportOrganizationAuditLog.ok).toBeNull();
+    expect(exportAuditLogs.rawBody.data?.exportOrganizationAuditLog.error).toBeDefined();
+    expect(exportAuditLogs.rawBody.data?.exportOrganizationAuditLog.error?.message).toBe(
+      'Unauthorized: You are not authorized to perform this action',
+    );
+  },
+);
+
+test.concurrent('Try to export Audit Logs from an Organization with authorized user', async () => {
+  const { createOrg, ownerToken } = await initSeed().createOwner();
+  const firstOrg = await createOrg();
+  await waitFor(5000);
+
+  const exportAuditLogs = await execute({
+    document: ExportAllAuditLogs,
+    variables: {
+      selector: {
+        organizationSlug: firstOrg.organization.id,
+      },
+      filter: {
+        startDate: '1960-12-31T22:00:00.000Z',
+        endDate: '2023-12-31T22:00:00.000Z',
+      },
+    },
+    token: ownerToken,
+  });
+
+  const url = exportAuditLogs.rawBody.data?.exportOrganizationAuditLog.ok?.url as string;
+  const parsedUrl = new URL(url);
+  const pathParts = parsedUrl.pathname.split('/');
+  const bucketName = pathParts[1];
+  const key = pathParts.slice(2).join('/');
+  const getObjectCommand = new GetObjectCommand({
+    Bucket: bucketName,
+    Key: key,
+  });
+  const result = await s3Client.send(getObjectCommand);
+  const bodyStream = await result.Body?.transformToString();
+  const csvToJSON = bodyStream?.split('\n').map(line => {
+    const [id, eventTime, organizationSlug, record] = line.split(',');
+    return { id, eventTime, organizationSlug, record };
+  });
+
+  expect(
+    csvToJSON?.find((log: any) => log.organizationSlug === 'ORGANIZATION_CREATED'),
+  ).toBeDefined();
+  expect(
+    csvToJSON?.find(
+      (log: any) => log.id === `Hive Audit Logs for Organization: ${firstOrg.organization.id}`,
+    ),
+  ).toBeDefined();
```

Comment:
@TuvalSimha This test does not give me any information on how the log is formatted or looks like.

I want to see a snapshot/assertions on the raw CSV string. Not the parsed CSV.